xml/dhcp_vendor_class.xml

```diff
@@ -0,0 +1,215 @@
+    <param pos="0" name="hw.device" value="Printer"/>
+    <param pos="0" name="hw.vendor" value="HP"/>
+    <param pos="1" name="hw.family"/>
+    <param pos="2" name="hw.model"/>
+    <param pos="0" name="hw.product" value="{hw.family} {hw.model}"/>
+    <param pos="0" name="os.vendor" value="HP"/>
+  </fingerprint>
+
+  <fingerprint pattern="^(?:Hewlett-Packard|HP) (OfficeJet|LaserJet|Printer|JetDirect)">
+    <description>HP Printer</description>
+    <example hw.family="LaserJet">Hewlett-Packard LaserJet</example>
+    <example hw.family="OfficeJet">Hewlett-Packard OfficeJet</example>
+    <example hw.family="LaserJet">HP LaserJet</example>
+    <example hw.family="Printer">HP Printer</example>
+    <example>Hewlett-Packard JetDirect</example>
+    <param pos="0" name="hw.device" value="Printer"/>
+    <param pos="0" name="hw.vendor" value="HP"/>
+    <param pos="1" name="hw.family"/>
+    <param pos="0" name="os.vendor" value="HP"/>
+  </fingerprint>
+
+  <fingerprint pattern="^Mfg=LEXMARK;Typ=(?:MFP|Printer);Mod=Lexmark (\S+);Ser=">
+    <description>Lexmark Printer</description>
+    <example hw.model="MX410de">Mfg=LEXMARK;Typ=MFP;Mod=Lexmark MX410de;Ser=70155PLM0Y5YD;</example>
+    <example hw.model="MS310dn">Mfg=LEXMARK;Typ=Printer;Mod=Lexmark MS310dn;Ser=451432LM0X7XY;</example>
+    <param pos="0" name="hw.device" value="Printer"/>
+    <param pos="0" name="hw.vendor" value="Lexmark"/>
+    <param pos="1" name="hw.model"/>
+    <param pos="0" name="os.vendor" value="Lexmark"/>
+  </fingerprint>
+
+  <fingerprint pattern="^Canon iR-ADV (C?\d+ ?\S*)">
+    <description>Canon imageRunner Printer</description>
+    <example hw.model="C5535 III">Canon iR-ADV C5535 III</example>
+    <example hw.model="C350">Canon iR-ADV C350</example>
+    <example hw.model="4545 III">Canon iR-ADV 4545 III</example>
+    <example hw.model="525">Canon iR-ADV 525</example>
+    <param pos="0" name="hw.device" value="Printer"/>
+    <param pos="0" name="hw.vendor" value="Canon"/>
+    <param pos="0" name="hw.family" value="imageRunner"/>
+    <param pos="1" name="hw.model"/>
+    <param pos="0" name="hw.product" value="{hw.family} {hw.model}"/>
+    <param pos="0" name="os.vendor" value="Canon"/>
+  </fingerprint>
+
+  <fingerprint pattern="Canon (D\d+) Series">
+    <description>Canon imageClas Printer</description>
+    <example hw.model="D1600">Canon D1600 Series</example>
+    <param pos="0" name="hw.device" value="Printer"/>
+    <param pos="0" name="hw.vendor" value="Canon"/>
+    <param pos="0" name="hw.family" value="imageClass"/>
+    <param pos="1" name="hw.model"/>
+    <param pos="0" name="hw.product" value="{hw.family} {hw.model}"/>
+    <param pos="0" name="os.vendor" value="Canon"/>
+  </fingerprint>
+
+  <fingerprint pattern="^Polycom-(VVX\S+)">
+    <description>Polycom IP Phone</description>
+    <example hw.model="VVX410">Polycom-VVX410</example>
```

Comment:
In this case please go ahead and set `hw.product` to be the same capture you are using for `hw.model`. That will enable auto population of the CPE data and result in values such as

```text
h:polycom:vvx400
h:polycom:vvx401
h:polycom:vvx410
h:polycom:vvx411
h:polycom:vvx450
h:polycom:vvx500
h:polycom:vvx501
h:polycom:vvx600
h:polycom:vvx601
```